### PR TITLE
Add tox.ini file for Flake8

### PR DESCRIPTION
**Pull request by squirrelo**

### CODE REVIEW

**Comment by wasade** [comment]
why drive flake8 via tox instead of travis?

**Comment by squirrelo** [comment]
because other linter plugins (such as for SublimeText) look for that file and can auto-set themselves using it.

**Comment by squirrelo** [comment]
The other filename it looks for is `setup.cfg` so can move to that if wanted.

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 95
```
Comment (wasade) [inline tox.ini]: please leave this at 80

Comment (squirrelo) [inline tox.ini]: :( ok.

Comment (ElDeveloper) [inline tox.ini]: ping

**Comment by squirrelo** [comment]
Change made

**Pull request merged**